apps/freeablo/farender/fontinfo.h

```diff
@@ -2,25 +2,46 @@
 #include "fa_nuklear.h"
 
 #include <array>
+#include <SDL_surface.h>
```

Comment:
I'd prefer to avoid referencing SDL directly outside of the Render component. Could whatever this is being sued for be wrapped in some way in there?

Reply:
Funny enough I think the easiest way to fix it also happens to be the most proper one. I just need to read correct character widths from `.bin` files, I used SDL surface to calculate them.

Reply:
Fixed. Plus added mouse interactions.